hepdata/modules/records/utils/data_files.py

```diff
@@ -387,37 +387,41 @@ def _move_files_for_record(rec_id):  # pragma: no cover
 
     os.makedirs(new_path, exist_ok=True)
 
+    is_sandbox = hep_submissions[0].status == 'sandbox'
```

Comment:
This gives `AttributeError: 'HEPSubmission' object has no attribute 'status'`.  It should be `overall_status`.

Maybe also check above in the line `inspire_id = hep_submissions[0].inspire_id` that `hep_submissions` is not an empty list, since it gives `IndexError: list index out of range`.  I triggered this exception when using the `-r` option with an invalid record ID.